ccm_web/client/src/models/feature.ts

```diff
@@ -9,12 +9,14 @@ interface FeatureDataProps {
   helpURLEnding: string
 }
 
+const adminRoles = [RoleEnum['Subaccount admin'], RoleEnum['Account Admin'], RoleEnum['Support Consultant']]
+
```

Comment:
add type `const adminRole: : RoleEnum[]`?

Reply:
TypeScript already knows that is the variable's type, so I guess you are just saying to guard against future changes by adding the type?

Reply:
Yes.

Reply:
You added these.